### PR TITLE
Update custom certs docs

### PR DESCRIPTION
**Pull request by abrennan89**
Fixes https://github.com/knative/docs/issues/4881

@dprotaso I took a stab at including the info about secrets but wasn't sure about it so please feel free to add comments or corrections. Thanks.

### CODE REVIEW

**Comment by abrennan89** [comment]
cc @knative/serving-wg-leads @knative/serving-reviewers

File: docs/serving/tag-resolution.md [hunk 1]
```diff
@@ -8,37 +8,46 @@ helps to provide consistency for Deployments. For more information, see the docu
 
 ## Custom certificates
 
-If you are using a registry that has a self-signed certificate, you must configure the Knative Serving controller to trust that certificate.
+If you are using a registry that has a self-signed certificate, you must configure the default Knative Serving `controller` Deployment to trust that certificate. You can configure trusting certificates by mounting your certificates into the `controller` Deployment, and then setting the environment variable appropriately.
```
Comment (skonto) [inline docs/serving/tag-resolution.md:11]: I think this should be aligned with the [operator's section](https://knative.dev/docs/install/operator/configuring-serving-cr/#ssl-certificate-for-controller.). We should describe both config map and secret based custom certs. The knative operator essentially modifies the controller deployment when the CR contains a custom-certs field and sets `SSL_CERT_DIR`.

Comment (abrennan89) [reply docs/serving/tag-resolution.md:11]: Is this required to complete the work requested by https://github.com/knative/docs/issues/4881 though?

I would like to be careful to keep this within scope of what was asked for in that issue vs delving too much into adding additional content or changing docs structure - if there's something missing that isn't part of that original issue, it should be opened as a new issue.

Comment (dprotaso) [reply docs/serving/tag-resolution.md:11]: Yeah I'd probably open an separate issue

**Comment by abrennan89** [comment]
@dprotaso PTAL 🙂

**Pull request merged**